From 23b5015a6976cb1134343703b393f2771a670d9c Mon Sep 17 00:00:00 2001
From: BambooIV <ningdisheng@gmail.com>
Date: Tue, 2 Feb 2016 16:38:32 +0800
Subject: [PATCH 17/50] nx503a: arm/dt: msm8974: Add support for voice svc
 driver

cherry-pick from 8a63ce79b2dbf5889df74ea3cc1c70ecf6323348

Add a new device that can be used by the voice svc
driver to allow userspace clients to send IOCTL
commands

Signed-off-by: BambooIV <ningdisheng@gmail.com>
---
 arch/arm/boot/dts/NX503A/msm8974.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/NX503A/msm8974.dtsi b/arch/arm/boot/dts/NX503A/msm8974.dtsi
index b139514..b049d63 100755
--- a/arch/arm/boot/dts/NX503A/msm8974.dtsi
+++ b/arch/arm/boot/dts/NX503A/msm8974.dtsi
@@ -1798,6 +1798,10 @@
 		compatible = "qti,msm-pcm-loopback";
 	};
 
+	qcom,msm-voice-svc {
+		compatible = "qcom,msm-voice-svc";
+	};
+
 	qcom,msm-dai-q6 {
 		compatible = "qcom,msm-dai-q6";
 		qcom,msm-dai-q6-sb-0-rx {
-- 
1.9.1

